sway/config/input.c

```diff
@@ -117,10 +117,13 @@ void merge_input_config(struct input_config *dst, struct input_config *src) {
 		free(dst->xkb_file);
 		dst->xkb_file = src->xkb_file ? strdup(src->xkb_file) : NULL;
 		dst->xkb_file_is_set = dst->xkb_file != NULL;
+		dst->xkb_layout = NULL;
```

Comment:
`xkb_file` overrides not just `xkb_layout`, but also `xkb_model`, `xkb_options`, `xkb_rules`, and `xkb_variant`.

Reply:
Yes but I would have assumed that these other options don't take effect if `xkb_file` is set and `xkb_layout` isn't. Looking at `sway_keyboard_compile_keymap` that does seem to be the case?

In any case I think it would be nice to log or display an error in case conflicting xkb_ options are specified in the same config block. But I'd defer that to another PR.